packages/lit-dev-content/samples/tutorials/intro-to-lit/manifest.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
WDYT about `tutorial.json`? Just makes it a little more obvious what it's for, vs `manifest` which is a little generic.